org.eclipse.iot.fourdiac.Ide.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "org.eclipse.iot.fourdiac.Ide",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk10"
+    ],
+    "command": "/app/4diac-ide/4diac-ide",
+    "modules": [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [ "/usr/lib/sdk/openjdk10/install.sh" ]
+        },
+        {
+            "name": "4diac-ide",
```

Comment:
can this not be built from source? Or is there a specific reason you need to use an upstream binary?

Reply:
In theory yes. But this is the official binary, signed with certificates from the Eclipse Foundation, which cannot be signed outside Eclipse infrastructure.

Aside from that I would need to replicate a good portion of the Eclipse IDE build, which I honestly am not capable of.

Reply:
the official binary thing makes sense. In this case I'd like to see that noted in the appdata description as well (see apps like VS Code which repackage upstream binaries)

Reply:
Ok, I add a paragraph about that.